lib/matplotlib/gridspec.py

```diff
@@ -232,38 +232,42 @@ def _normalize(key, size, axis):  # Includes last index.
 
 class GridSpec(GridSpecBase):
     """
-    Specifies the geometry of the grid that a subplot can be placed in.
+    A grid layout to place subplots within a figure.
 
-    The location of grid is determined by similar way as the SubplotParams.
+    The location of the grid cells is determined in a similar way as
+    `~.figure.SubplotParams` using *left*, *right*, *top*, *bottom*, *wspace*
+    and *hspace*.
     """
     def __init__(self, nrows, ncols, figure=None,
                  left=None, bottom=None, right=None, top=None,
                  wspace=None, hspace=None,
                  width_ratios=None, height_ratios=None):
         """
-        The number of rows and number of columns of the grid need to be set.
-        Optionally, the subplot layout parameters (e.g., left, right, etc.)
-        can be tuned.
-
         Parameters
         ----------
         nrows, ncols : int
             The number of rows and columns of the grid.
 
         figure : `~.figure.Figure`, optional
+            Only used for constrained layout to create a proper layoutbox.
```

Comment:
This is better than nothing, but still not too helpful.

@jklymak Maybe you have a better description?